Assets/MRTK/Tools/MSBuild/Scripts/TargetFramework.cs

```diff
@@ -35,7 +35,11 @@ public static string AsMSBuildString(this TargetFramework @this)
                 case TargetFramework.Net20:
                     return "net20";
                 case TargetFramework.Net46:
```

Comment:
Do we need to add a new entry in this enum for `471`?

Reply:
Good catch. We might.

Reply:
Unity has its own ApiCompatibilityLevel enum and there is not an entry for 471 in Unity 2020. So technically from Unity's standpoint we are still using .net 4.6 and the reason I added 471 here is during msbuild build time the compiler will complain as some of Unity's dependency DLLs is targeting 471.